src/warnet/tank.py

```diff
@@ -91,8 +94,9 @@ def parse_graph_node(self, node):
                 "impl": node["ln"],
                 "ln_image": node.get("ln_image", "lightninglabs/lnd:v0.18.0-beta"),
                 "cb_image": node.get("ln_cb_image", None),
-                "ln_config": node.get("ln_config", "")
+                "ln_config": node.get("ln_config", ""),
             }
+            from warnet.lnnode import LNNode
```

Comment:
Is the idea to do lazy loading? Are we saving a lot by doing this?

Reply:
Ah, I think this has spilled over from another change I was making but didn't include here! The idea though is to avoid recursive imports, not lazy loading.

The other change I wanted to make, was to split out (dependencies) for the CLI fully from the server. Then you could `pip install` one or the other, or both. IIRC `cli/rpc.py` was importing a single constant from `warnet/server.py`, which ended up in cli importing everything. I don't remember exactly how now, but this was part of mitigating that (in the least effort way).

IMO it's fine to keep in here, but I can remove it if you'd prefer

Reply:
I'd prefer it taken out b/c it doesn't relate to removing compose.

Reply:
Done in 9693a42

Reply:
It appears this was needed here: https://github.com/bitcoin-dev-project/warnet/actions/runs/9530483126/job/26270408101?pr=382#step:6:245

Going to revert

Reply:
I see!

It looks like lnnode was using the abstract class to abstract away the backend which may have avoided the circular dependency. I'm generally in favor of not having the abstract class since we now only have one backend; however, in terms of these imports, the indirection the abc afforded us seems pretty sweet.

Reply:
Anyway, if others ask for bringing back the backend abc, then I think we should unwind these late loading imports. If others don't care about the backend abc, then I'm fine with this.

Reply:
Correct.

I was working on it in this branch: https://github.com/bitcoin-dev-project/warnet/commit/6c1979bd741e98c388b6ad929e62ab872eacb72d which, unfortunately due to our current structure also turned out to be waaaay bigger than I expected. I may take a go at it again if we merge this one though as I think it will be significantly smaller. IIRC the main culprits were doing things like `from warnet import WARNET_SERVER_PORT` in some file which really did not need to be importing everything from server (cli). etc.

Reply:
Neat. Thank you for showing me that.